docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -0,0 +1,606 @@
+        END LOOP;
+    END;
+    /
+
+    --enable referential constraints
+
+    BEGIN
+        FOR c IN (SELECT table_name, constraint_name
+                FROM user_constraints
+                WHERE constraint_type IN ('R') AND OWNER = '<SCHEMA_NAME>' )
+        LOOP
+           EXECUTE IMMEDIATE 'ALTER TABLE ' || c.table_name || ' ENABLE CONSTRAINT ' || c.constraint_name;
+        END LOOP;
+    END;
+    /
+    ```
+
+1. Verify your migration. After the schema and data import is complete, the automated part of the database migration process is considered complete. You should manually run validation queries on both the source and target databases to ensure that the data is correctly migrated. A sample query to validate the databases can include checking the row count of each table.
+
+### End migration
+
+To end the migration, you need to clean up the export directory (export-dir), and Voyager state ( Voyager-related metadata) stored in the target YugabyteDB database and source database.
+
+Run the `yb-voyager end migration` command to perform the clean up, and to back up the schema, data, migration reports, and log files by providing the backup related flags (mandatory) as follows:
+
+```sh
+# Replace the argument values with those applicable for your migration.
+yb-voyager end migration --export-dir <EXPORT_DIR> \
+        --backup-log-files <true, false, yes, no, 1, 0> \
+        --backup-data-files <true, false, yes, no, 1, 0> \
+        --backup-schema-files <true, false, yes, no, 1, 0> \
+        --save-migration-reports <true, false, yes, no, 1, 0> \
+        # Set optional argument to store a back up of any of the above arguments.
+        --backup-dir <BACKUP_DIR>
+```
+
+Note that after you end the migration, you will _not_ be able to continue further. If you want to back up the schema, data, log files, and the migration reports (`analyze-schema` report and `get data-migration-report` output) for future reference, the command provides an additional argument `--backup-dir`, using which you can pass the path of the directory where the backup content needs to be saved (based on what you choose to back up).
+
+Refer to [end migration](../../reference/end-migration/) for more details on the arguments.
+
+## Limitations
+
+In addition to the Live migration [limitations](../live-migrate/#limitations), the following additional limitations apply to the fall-back feature:
+
+1. Fall-back is unsupported with a YugabyteDB cluster running on YugabyteDB Managed.
+1. SSL Connectivity is unsupported for export or streaming events from YugabyteDB during `export data from target`.
+1. In the fall-back phase, you need to manually disable (and subsequently re-enable if required) constraints/indexes/triggers on the source database.
+1. yb-voyager provides limited datatypes support with YugabyteDB CDC during `export data from target` for datatypes such as DECIMAL and Timestamp.
```

Comment:
Not sure what this means
- support for data types during CDC is limited, eg, Decimal and Timestamp are not supported; OR
- support for Decimal and Timestamp data types is limited during CDC; 
- or something else entirely

Reply:
cc @priyanshi-yb - I think its the first point in the suggestion. Can you clarify?

Reply:
yes 1st one is correct.